qemu/tests/multi_vms_with_stress.py

```diff
@@ -95,6 +96,13 @@ def wait_for_shutdown_all_vms(vms, sessions):
     run_stress_background()
     is_stress_alive()
 
+    if params.get('set_maxcpus'):
+        online_cpu = cpu.online_count() * 2 // 5
+        if (online_cpu % 2) != 0:
+            online_cpu += 1
```

Comment:
online_cpu=1 if online_cpu==1 else online_cpu -1 

the total avoid overcommit ?

Reply:
After offline discussion, the question has been solved